cmd/grafana-app-sdk/project_local.go

```diff
@@ -47,6 +47,7 @@ type localEnvConfig struct {
 	GenerateGrafanaDeployment bool                  `json:"generateGrafanaDeployment" yaml:"generateGrafanaDeployment"`
 	GrafanaImage              string                `json:"grafanaImage" yaml:"grafanaImage"`
 	GrafanaInstallPlugins     string                `json:"grafanaInstallPlugins" yaml:"grafanaInstallPlugins"`
+	GrafanaAnonymousAuth      bool                  `json:"grafanaAnonymousAuth" yaml:"grafanaAnonymousAuth"`
```

Comment:
nit: we could call this `GrafanaWithAnonymousAuth` to be compatible with our deployment tools. see [search](https://github.com/search?q=repo%3Agrafana%2Fdeployment_tools%20WithAnony&type=code)